### ISSUE TITLE
Flutter desktop App gets crashed on first launch in windows 7 

### ISSUE THREAD

**Issue by masoodtamboli** [flutter/flutter#73233]
I built a sample app in windows 7 with flutter for windows and it gets crashed on first launch.
Here are logs of program 

`F:\Masood\my_app\my_app>flutter run -d windows
Launching lib\main.dart on Windows in debug mode...
Building Windows application...
EGL: Failed to initializeEGL: Failed to initializeEGL: Failed to initializeEGL: Failed to initializeEGL: Failed to choose first context
[FATAL:flutter/shell/gpu/gpu_surface_gl_delegate.cc(50)] Check failed: gl_version_string. The GL proc resolver's glGetString(GL_VERSION) failed
Error connecting to the service protocol: _flutter.listViews: (-32000) Bad state: The client closed with pending request "_flutter.listViews".`

**flutter doctor -v output**

[√] Flutter (Channel dev, 1.26.0-1.0.pre, on Microsoft Windows [Version 6.1.7601], locale en-US)
    • Flutter version 1.26.0-1.0.pre at C:\Dev\flutter
    • Framework revision 63062a6443 (3 weeks ago), 2020-12-13 23:19:13 +0800
    • Engine revision 4797b06652
    • Dart version 2.12.0 (build 2.12.0-141.0.dev)

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Users\hasnain\AppData\Local\Android\Sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: C:\Dev\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.8.3)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.8.30804.86
    • Windows 10 SDK version 10.0.17763.0

[√] Android Studio (version 4.1.0)
    • Android Studio at C:\Dev\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] VS Code (version 1.52.1)
    • VS Code at C:\Users\hasnain\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.18.0

[√] Connected device (1 available)
    • Windows (desktop) • windows • windows-x64 • Microsoft Windows [Version 6.1.7601]

• No issues found!

**Outputs of flutter run -v**

`[  +87 ms] executing: [C:\Dev\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[  +70 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ] 63062a64432cce03315d6b5196fda7912866eb37
[   +1 ms] executing: [C:\Dev\flutter/] git tag --points-at 63062a64432cce03315d6b5196fda7912866eb37
[  +39 ms] Exit code 0 from: git tag --points-at 63062a64432cce03315d6b5196fda7912866eb37
[   +1 ms] 1.26.0-1.0.pre
[  +68 ms] executing: [C:\Dev\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +24 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/dev
[        ] executing: [C:\Dev\flutter/] git ls-remote --get-url origin
[  +63 ms] Exit code 0 from: git ls-remote --get-url origin
[   +4 ms] https://github.com/flutter/flutter.git
[  +96 ms] executing: [C:\Dev\flutter/] git rev-parse --abbrev-ref HEAD
[  +28 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +1 ms] dev
[  +93 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +3 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +105 ms] executing: C:\Users\hasnain\AppData\Local\Android\Sdk\platform-tools\adb.exe devices -l
[  +35 ms] List of devices attached
[  +29 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +4 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +187 ms] Skipping pub get: version match.
[ +297 ms] Found plugin integration_test at C:\Dev\flutter\packages\integration_test\
[ +358 ms] Found plugin integration_test at C:\Dev\flutter\packages\integration_test\
[  +18 ms] Generating F:\Masood\my_app\my_app\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[ +164 ms] Initializing file store
[  +10 ms] Skipping target: gen_localizations
[   +6 ms] complete
[   +7 ms] Launching lib\main.dart on Windows in debug mode...
[  +10 ms] C:\Dev\flutter\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev
C:\Dev\flutter\bin\cache\artifacts\engine\windows-x64\frontend_server.dart.snapshot --sdk-root
C:\Dev\flutter\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --incremental --target=flutter --debugger-module-names
--experimental-emit-debug-metadata --output-dill C:\Users\hasnain\AppData\Local\Temp\flutter_tools.33e7d99\flutter_tool.4f667730\app.dill --packages
F:\Masood\my_app\my_app\.dart_tool\package_config.json -Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts --track-widget-creation
--filesystem-scheme org-dartlang-root --initialize-from-dill build\cache.dill.track.dill
[  +41 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16 -require
Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64 Microsoft.VisualStudio.Component.VC.CMake.Project
[  +19 ms] Exit code 0 from: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16
-requires Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64
Microsoft.VisualStudio.Component.VC.CMake.Project
[   +5 ms] [
             {
               "instanceId": "612ab6a3",
               "installDate": "2020-12-08T08:51:14Z",
               "installationName": "VisualStudio/16.8.3+30804.86",
               "installationPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community",
               "installationVersion": "16.8.30804.86",
               "productId": "Microsoft.VisualStudio.Product.Community",
               "productPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\Common7\\IDE\\devenv.exe",
               "state": 8534390167962648575,
               "isComplete": true,
               "isLaunchable": true,
               "isPrerelease": false,
               "isRebootRequired": false,
               "displayName": "Visual Studio Community 2019",
               "description": "Powerful IDE, free for students, open-source contributors, and individuals",
               "channelId": "VisualStudio.16.Release",
               "channelUri": "https://aka.ms/vs/16/release/channel",
               "enginePath": "C:\\Program Files (x86)\\Microsoft Visual
               Studio\\Installer\\resources\\app\\ServiceHub\\Services\\Microsoft.VisualStudio.Setup.Service",
               "releaseNotes": "https://go.microsoft.com/fwlink/?LinkId=660893#16.8.3",
               "thirdPartyNotices": "https://go.microsoft.com/fwlink/?LinkId=660909",
               "updateDate": "2020-12-12T19:15:26.8682986Z",
               "catalog": {
                 "buildBranch": "d16.8",
                 "buildVersion": "16.8.30804.86",
                 "id": "VisualStudio/16.8.3+30804.86",
                 "localBuild": "build-lab",
                 "manifestName": "VisualStudio",
                 "manifestType": "installer",
                 "productDisplayVersion": "16.8.3",
                 "productLine": "Dev16",
                 "productLineVersion": "2019",
                 "productMilestone": "RTW",
                 "productMilestoneIsPreRelease": "False",
                 "productName": "Visual Studio",
                 "productPatchVersion": "3",
                 "productPreReleaseMilestoneSuffix": "1.0",
                 "productSemanticVersion": "16.8.3+30804.86",
                 "requiredEngineVersion": "2.8.3267.30329"
               },
               "properties": {
                 "campaignId": "1158453142.1595257950",
                 "canceled": "0",
                 "channelManifestId": "VisualStudio.16.Release/16.8.3+30804.86",
                 "defaultProgram": "VisualStudio.612ab6a3",
                 "nickname": "",
                 "setupEngineFilePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\vs_installershell.exe"
               }
             }
           ]
[   +8 ms] Building Windows application...
[  +17 ms] <- compile package:my_app/main.dart
[   +4 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe -
F:\Masood\my_app\my_app\windows -B build\windows -G Visual Studio 16 2019
[+3545 ms] -- Selecting Windows SDK version 10.0.17763.0 to target Windows 6.1.7601.
[ +908 ms] -- Configuring done
[+2157 ms] -- Generating done
[ +130 ms] -- Build files have been written to: F:/Masood/my_app/my_app/build/windows
[  +66 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe
--build build\windows --config Debug --target INSTALL --verbose
[+5324 ms] Microsoft (R) Build Engine version 16.8.2+25e4d540b for .NET Framework
[        ] Copyright (C) Microsoft Corporation. All rights reserved.
[+2333 ms] Build started 1/2/2021 11:40:44 PM.
[+5093 ms] Project "F:\Masood\my_app\my_app\build\windows\INSTALL.vcxproj" on node 1 (default targets).
[        ] Project "F:\Masood\my_app\my_app\build\windows\INSTALL.vcxproj" (1) is building "F:\Masood\my_app\my_app\build\windows\ZERO_CHECK.vcxproj"
(2) on node 1 (default targets).
[   +1 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[ +400 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[ +280 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\ZERO_CHECK.lastbuildstate".
[   +9 ms] Done Building Project "F:\Masood\my_app\my_app\build\windows\ZERO_CHECK.vcxproj" (default targets).
[ +280 ms] Project "F:\Masood\my_app\my_app\build\windows\INSTALL.vcxproj" (1) is building "F:\Masood\my_app\my_app\build\windows\ALL_BUILD.vcxproj" (
on node 1 (default targets).
[   +1 ms] Project "F:\Masood\my_app\my_app\build\windows\ALL_BUILD.vcxproj" (3) is building
"F:\Masood\my_app\my_app\build\windows\flutter\flutter_wrapper_app.vcxproj" (4) on node 1 (default targets).
[   +1 ms] Project "F:\Masood\my_app\my_app\build\windows\flutter\flutter_wrapper_app.vcxproj" (4) is building
"F:\Masood\my_app\my_app\build\windows\flutter\flutter_assemble.vcxproj" (5) on node 1 (default targets).
[  +25 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +66 ms] CustomBuild:
[   +5 ms]   "The build of 'F:\Masood\my_app\my_app\build\windows\CMakeFiles\4c704b714a40d7fced27fda7ad7312a3\flutter_assemble.rule' depends on
'F:\MASOOD\MY_APP\MY_APP\WINDOWS\FLUTTER\EPHEMERAL\FLUTTER_WINDOWS.DLL' which is produced by the build of
'F:\Masood\my_app\my_app\build\windows\CMakeFiles\d4eb395f1eaf9a7d75c75644cd6360f3\flutter_windows.dll.rule'. The items cannot be built in parallel."
[+2555 ms]   Generating F:/Masood/my_app/my_app/windows/flutter/ephemeral/flutter_windows.dll,
F:/Masood/my_app/my_app/windows/flutter/ephemeral/flutter_export.h, F:/Masood/my_app/my_app/windows/flutter/ephemeral/flutter_windows.h,
F:/Masood/my_app/my_app/windows/flutter/ephemeral/flutter_messenger.h, F:/Masood/my_app/my_app/windows/flutter/ephemeral/flutter_plugin_registrar.h,
F:/Masood/my_app/my_app/windows/flutter/ephemeral/cpp_client_wrapper/core_implementations.cc,
F:/Masood/my_app/my_app/windows/flutter/ephemeral/cpp_client_wrapper/standard_codec.cc,
F:/Masood/my_app/my_app/windows/flutter/ephemeral/cpp_client_wrapper/plugin_registrar.cc,
F:/Masood/my_app/my_app/windows/flutter/ephemeral/cpp_client_wrapper/flutter_engine.cc,
F:/Masood/my_app/my_app/windows/flutter/ephemeral/cpp_client_wrapper/flutter_view_controller.cc, _phony_
[+13841 ms]   [  +88 ms] executing: [C:\Dev\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[+1407 ms]   [+1450 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +2 ms]   [   +2 ms] 63062a64432cce03315d6b5196fda7912866eb37
[        ]   [   +1 ms] executing: [C:\Dev\flutter/] git tag --points-at 63062a64432cce03315d6b5196fda7912866eb37
[+2402 ms]   [+2402 ms] Exit code 0 from: git tag --points-at 63062a64432cce03315d6b5196fda7912866eb37
[        ]   [        ] 1.26.0-1.0.pre
[  +63 ms]   [  +64 ms] executing: [C:\Dev\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +86 ms]   [  +85 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[  +13 ms]   [        ] origin/dev
[   +1 ms]   [        ] executing: [C:\Dev\flutter/] git ls-remote --get-url origin
[  +24 ms]   [  +39 ms] Exit code 0 from: git ls-remote --get-url origin
[   +1 ms]   [        ] https://github.com/flutter/flutter.git
[ +118 ms]   [ +118 ms] executing: [C:\Dev\flutter/] git rev-parse --abbrev-ref HEAD
[  +41 ms]   [  +42 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +1 ms]   [        ] dev
[ +207 ms]   [ +208 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[  +16 ms]   [  +17 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[+2185 ms]   [+2177 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'PubDependencies' is not required, skipping update.
[ +162 ms]   [ +179 ms] Initializing file store
[  +56 ms]   [  +56 ms] Done initializing file store
[ +221 ms]   [ +220 ms] Skipping target: gen_localizations
[+8320 ms]   [+8318 ms] Skipping target: unpack_windows
[+5221 ms]   [+5222 ms] Skipping target: kernel_snapshot
[+1406 ms]   [+1405 ms] Skipping target: debug_bundle_windows_assets
[        ]   [   +1 ms] Persisting file store
[  +81 ms]   [  +82 ms] Done persisting file store
[  +28 ms]   [  +28 ms] build succeeded.
[ +103 ms]   [ +104 ms] "flutter assemble" took 17,751ms.
[ +261 ms]   [ +260 ms] ensureAnalyticsSent: 255ms
[   +3 ms]   [   +3 ms] Running shutdown hooks
[   +1 ms]   [        ] Shutdown hooks complete
[   +1 ms]   [   +1 ms] exiting with code 0
[ +488 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "x64\Debug\flutter_assemble\flutter_assemble.tlog\flutter_assemble.lastbuildstate".
[        ] Done Building Project "F:\Masood\my_app\my_app\build\windows\flutter\flutter_assemble.vcxproj" (default targets).
[ +142 ms] InitializeBuildStatus:
[   +1 ms]   Creating "flutter_wrapper_app.dir\Debug\flutter_.AC502B6C.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +68 ms] CustomBuild:
[   +1 ms]   All outputs are up-to-date.
[ +540 ms] ClCompile:
[        ]   All outputs are up-to-date.
[ +254 ms] Lib:
[   +1 ms]   All outputs are up-to-date.
[  +63 ms]   flutter_wrapper_app.vcxproj -> F:\Masood\my_app\my_app\build\windows\flutter\Debug\flutter_wrapper_app.lib
[  +71 ms] FinalizeBuildStatus:
[        ]   Deleting file "flutter_wrapper_app.dir\Debug\flutter_.AC502B6C.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "flutter_wrapper_app.dir\Debug\flutter_.AC502B6C.tlog\flutter_wrapper_app.lastbuildstate".
[   +1 ms] Done Building Project "F:\Masood\my_app\my_app\build\windows\flutter\flutter_wrapper_app.vcxproj" (default targets).
[ +583 ms] Project "F:\Masood\my_app\my_app\build\windows\ALL_BUILD.vcxproj" (3) is building
"F:\Masood\my_app\my_app\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (6) on node 1 (default targets).
[  +34 ms] InitializeBuildStatus:
[  +18 ms]   Creating "flutter_wrapper_plugin.dir\Debug\flutter_.56295CB0.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +2 ms] CustomBuild:
[   +2 ms]   All outputs are up-to-date.
[  +33 ms] ClCompile:
[        ]   All outputs are up-to-date.
[  +76 ms] Lib:
[   +1 ms]   All outputs are up-to-date.
[   +8 ms]   flutter_wrapper_plugin.vcxproj -> F:\Masood\my_app\my_app\build\windows\flutter\Debug\flutter_wrapper_plugin.lib
[   +9 ms] FinalizeBuildStatus:
[        ]   Deleting file "flutter_wrapper_plugin.dir\Debug\flutter_.56295CB0.tlog\unsuccessfulbuild".
[        ]   Touching "flutter_wrapper_plugin.dir\Debug\flutter_.56295CB0.tlog\flutter_wrapper_plugin.lastbuildstate".
[        ] Done Building Project "F:\Masood\my_app\my_app\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (default targets).
[ +193 ms] Project "F:\Masood\my_app\my_app\build\windows\ALL_BUILD.vcxproj" (3) is building
"F:\Masood\my_app\my_app\build\windows\runner\my_app.vcxproj" (7) on node 1 (default targets).
[   +2 ms] InitializeBuildStatus:
[        ]   Creating "my_app.dir\Debug\my_app.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +78 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[ +272 ms] ClCompile:
[        ]   C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29333\bin\HostX64\x64\CL.exe /c
/IF:\Masood\my_app\my_app\windows /IF:\Masood\my_app\my_app\windows\flutter\ephemeral
/IF:\Masood\my_app\my_app\windows\flutter\ephemeral\cpp_client_wrapper\include /Zi /nologo /W4 /WX /diagnostics:column /Od /Ob0 /D WIN32 /D _WINDOWS /
_HAS_EXCEPTIONS=0 /D _DEBUG /D NOMINMAX /D UNICODE /D _UNICODE /D "CMAKE_INTDIR=\"Debug\"" /D _UNICODE /D UNICODE /Gm- /EHsc /RTC1 /MDd /GS /fp:precis
/Zc:wchar_t /Zc:forScope /Zc:inline /GR /std:c++17 /Fo"my_app.dir\Debug\\" /Fd"my_app.dir\Debug\vc142.pdb" /Gd /TP /wd4100 /errorReport:queue
F:\Masood\my_app\my_app\windows\flutter\generated_plugin_registrant.cc F:\Masood\my_app\my_app\windows\runner\flutter_window.cpp
[+1115 ms]   generated_plugin_registrant.cc
[+2925 ms]   flutter_window.cpp
[+3914 ms]   Generating Code...
[ +949 ms] ResourceCompile:
[        ]   All outputs are up-to-date.
[+1165 ms] Link:
[        ]   C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29333\bin\HostX64\x64\link.exe /ERRORREPORT:QUEUE
/OUT:"F:\Masood\my_app\my_app\build\windows\runner\Debug\my_app.exe" /INCREMENTAL /NOLOGO ..\flutter\Debug\flutter_wrapper_app.lib
F:\Masood\my_app\my_app\windows\flutter\ephemeral\flutter_windows.dll.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib
oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTUAC:"level='asInvoker' uiAccess='false'" /manifest:embed
/manifestinput:F:\Masood\my_app\my_app\windows\runner\runner.exe.manifest /DEBUG /PDB:"F:/Masood/my_app/my_app/build/windows/runner/Debug/my_app.pdb"
/SUBSYSTEM:WINDOWS /TLBID:1 /DYNAMICBASE /NXCOMPAT /IMPLIB:"F:/Masood/my_app/my_app/build/windows/runner/Debug/my_app.lib" /MACHINE:X64  /machine:x64
my_app.dir\Debug\Runner.res
[   +2 ms]   my_app.dir\Debug\flutter_window.obj
[        ]   my_app.dir\Debug\main.obj
[        ]   my_app.dir\Debug\run_loop.obj
[        ]   my_app.dir\Debug\utils.obj
[        ]   my_app.dir\Debug\win32_window.obj
[        ]   my_app.dir\Debug\generated_plugin_registrant.obj
[+2161 ms]   my_app.vcxproj -> F:\Masood\my_app\my_app\build\windows\runner\Debug\my_app.exe
[  +84 ms] FinalizeBuildStatus:
[        ]   Deleting file "my_app.dir\Debug\my_app.tlog\unsuccessfulbuild".
[        ]   Touching "my_app.dir\Debug\my_app.tlog\my_app.lastbuildstate".
[        ] Done Building Project "F:\Masood\my_app\my_app\build\windows\runner\my_app.vcxproj" (default targets).
[  +28 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[ +175 ms] CustomBuild:
[        ]   All outputs are up-to-date.
[  +18 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\ALL_BUILD.lastbuildstate".
[   +1 ms] Done Building Project "F:\Masood\my_app\my_app\build\windows\ALL_BUILD.vcxproj" (default targets).
[  +38 ms] InitializeBuildStatus:
[   +2 ms]   Creating "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[ +315 ms] PostBuildEvent:
[        ]   setlocal
[        ]   "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe"
-DBUILD_TYPE=Debug -P cmake_install.cmake
[   +1 ms]   if %errorlevel% neq 0 goto :cmEnd
[        ]   :cmEnd
[        ]   endlocal & call :cmErrorLevel %errorlevel% & goto :cmDone
[        ]   :cmErrorLevel
[        ]   exit /b %1
[        ]   :cmDone
[        ]   if %errorlevel% neq 0 goto :VCEnd
[   +1 ms]   :VCEnd
[ +486 ms]   -- Install configuration: "Debug"
[  +28 ms]   -- Up-to-date: F:/Masood/my_app/my_app/build/windows/runner/Debug/data/icudtl.dat
[   +1 ms]   -- Up-to-date: F:/Masood/my_app/my_app/build/windows/runner/Debug/flutter_windows.dll
[ +217 ms]   -- Installing: F:/Masood/my_app/my_app/build/windows/runner/Debug/data/flutter_assets
[   +5 ms]   -- Installing: F:/Masood/my_app/my_app/build/windows/runner/Debug/data/flutter_assets/AssetManifest.json
[        ]   -- Installing: F:/Masood/my_app/my_app/build/windows/runner/Debug/data/flutter_assets/FontManifest.json
[        ]   -- Installing: F:/Masood/my_app/my_app/build/windows/runner/Debug/data/flutter_assets/fonts
[        ]   -- Installing: F:/Masood/my_app/my_app/build/windows/runner/Debug/data/flutter_assets/fonts/MaterialIcons-Regular.otf
[   +2 ms]   -- Installing: F:/Masood/my_app/my_app/build/windows/runner/Debug/data/flutter_assets/kernel_blob.bin
[  +98 ms]   -- Installing: F:/Masood/my_app/my_app/build/windows/runner/Debug/data/flutter_assets/NOTICES
[   +2 ms]   -- Installing: F:/Masood/my_app/my_app/build/windows/runner/Debug/data/flutter_assets/packages
[        ]   -- Installing: F:/Masood/my_app/my_app/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons
[   +1 ms]   -- Installing: F:/Masood/my_app/my_app/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets
[        ]   -- Installing: F:/Masood/my_app/my_app/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets/CupertinoIcons.ttf
[  +56 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild".
[   +3 ms]   Touching "x64\Debug\INSTALL\INSTALL.tlog\INSTALL.lastbuildstate".
[   +3 ms] Done Building Project "F:\Masood\my_app\my_app\build\windows\INSTALL.vcxproj" (default targets).
[ +113 ms] Build succeeded.
[  +14 ms]     0 Warning(s)
[        ]     0 Error(s)
[  +67 ms] Time Elapsed 00:01:02.01
[ +715 ms] Building Windows application... (completed in 77.2s)
[+4697 ms] EGL: Failed to initializeEGL: Failed to initializeEGL: Failed to initializeEGL: Failed to initializeEGL: Failed to choose first context
[ +411 ms] [FATAL:flutter/shell/gpu/gpu_surface_gl_delegate.cc(50)] Check failed: gl_version_string. The GL proc resolver's glGetString(GL_VERSION)
failed
[ +683 ms] Observatory URL on device: http://127.0.0.1:51088/D52ZrEbCFHc=/
[   +9 ms] Caching compiled dill
[ +448 ms] Connecting to service protocol: http://127.0.0.1:51088/D52ZrEbCFHc=/
[   +5 ms] Launching a Dart Developer Service (DDS) instance at http://127.0.0.1:0, connecting to VM service at http://127.0.0.1:51088/D52ZrEbCFHc=/.
[ +857 ms] DDS is listening at http://127.0.0.1:51092/VUICdRI51Rk=/.
[ +211 ms] Successfully connected to service protocol: http://127.0.0.1:51088/D52ZrEbCFHc=/
[+7321 ms] Error connecting to the service protocol: _flutter.listViews: (-32000) Bad state: The client closed with pending request
"_flutter.listViews".
[  +10 ms] "flutter run" took 93,235ms.
[  +16 ms]
           #0      throwToolExit (package:flutter_tools/src/base/common.dart:10:3)
           #1      RunCommand.runCommand (package:flutter_tools/src/commands/run.dart:623:7)
           <asynchronous suspension>
           #2      FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:1134:12)
           <asynchronous suspension>
           #3      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:986:27)
           <asynchronous suspension>
           #4      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #5      AppContext.run (package:flutter_tools/src/base/context.dart:149:12)
           <asynchronous suspension>
           #6      CommandRunner.runCommand (package:args/command_runner.dart:197:13)
           <asynchronous suspension>
           #7      FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:274:9)
           <asynchronous suspension>
           #8      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #9      AppContext.run (package:flutter_tools/src/base/context.dart:149:12)
           <asynchronous suspension>
           #10     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:230:5)
           <asynchronous suspension>
           #11     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:63:9)
           <asynchronous suspension>
           #12     run.<anonymous closure> (package:flutter_tools/runner.dart:61:12)
           <asynchronous suspension>
           #13     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #14     AppContext.run (package:flutter_tools/src/base/context.dart:149:12)
           <asynchronous suspension>
           #15     runInContext (package:flutter_tools/src/context_runner.dart:70:10)
           <asynchronous suspension>
           #16     main (package:flutter_tools/executable.dart:88:3)
           <asynchronous suspension>

[ +256 ms] ensureAnalyticsSent: 252ms
[   +2 ms] Running shutdown hooks
[   +1 ms] Shutdown hook priority 4
[   +9 ms] Shutdown hooks complete
[   +2 ms] exiting with code 2`

**Comment by stuartmorgan** [comment]
Duplicate of https://github.com/flutter/flutter/issues/73040

Could you provide the details of your graphics card and driver in that issue?

**Comment by masoodtamboli** [comment]
Graphics details
![1](https://user-images.githubusercontent.com/45930017/103518484-b1962e80-4e28-11eb-89af-112427f38c72.png)
